src/integrationTest/java/com/hivemq/helmcharts/LocalHelmChartDeploymentIT.java

```diff
@@ -0,0 +1,56 @@
+package com.hivemq.helmcharts;
+
+import com.hivemq.client.mqtt.MqttGlobalPublishFilter;
+import com.hivemq.client.mqtt.datatypes.MqttQos;
+import com.hivemq.client.mqtt.mqtt5.Mqtt5Client;
+import com.hivemq.helmcharts.util.OperatorHelmChartContainer;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Using locally build images tests that all the artifacts are installed and is possible to send a mqtt message
+ */
+@SuppressWarnings("DuplicatedCode")
+@Testcontainers
+public class LocalHelmChartDeploymentIT {
+
+    @Container
+    private final @NotNull OperatorHelmChartContainer
+            container = new OperatorHelmChartContainer("v1.23.4-k3s1",
+            "k3s.dockerfile",
+            "values/customTestValues.yaml")
+            .withLocalImages();
+
+    @Timeout(value = 3, unit = TimeUnit.MINUTES)
+    @Test
+    public void withLocalImages_mqttMessagePublishedReceived() throws Exception {
+        final var client = Mqtt5Client.builder()
+                .automaticReconnectWithDefaultConfig()
```

Comment:
same here, reconnect is not necessary

Reply:
Yes, same here, otherwise tests will not, work, it seems an issue between when the operator says the broker is ready and actually receives connections